onnx2kerastl/utils.py

```diff
@@ -21,8 +21,10 @@ def ensure_numpy_type(obj):
     """
     if is_numpy(obj):
         return obj
+    elif isinstance(obj, tf.Tensor):
+        return obj.numpy()
```

Comment:
are you sure about it? i think we wanted to ensure with this function something is a hyper parameter, where does it blocks you?